user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -27,3 +29,81 @@ Please note the following regarding suggestion events:
+
+- The [time to live](xref:Specifying_TTL_overrides) for trending is at least 1 day for real-time trending and 1 month for minute records.
+
+### Providing feedback
+
+<!--RN 39640-->
+
+To provide feedback:
+
+1. In the Alarm Console, open the *Anomalies* or *Suggestion events* tab. See [Navigating the light bulb menu](xref:Light_Bulb_Feature#navigating-the-light-bulb-menu) and [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+1. Verify that the ![Feedback](~/user-guide/images/Feedback_Column.png) column is visible. If it is not:
+
+   - Close the *Anomalies* or *Suggestion events* tab and reopen it. However, make sure not to use the *Reopen closed alarm lists* option. See [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+
+     This step might be necessary if you have recently upgraded from a version of DataMiner that did not include the Feedback column to one that does (DataMiner 10.4.11/10.5.0 or higher). Since Cube saves your open tabs along with their column layout, the new column will not automatically appear in tabs that were open before the upgrade.
+
+   - Ensure you are using DataMiner version 10.4.11/10.5.0 or higher, or confirm that the *AnomalyFeedback* option is included in the *SoftLaunchOptions.xml* file.
+
+   - If possible, manually add the column by right-clicking the column headers and selecting *Add or remove columns* > *Actions* > *Feedback*.
+
+1. Hover the mouse pointer over a suggestion event or alarm to view the ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) and ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons.
+
+   - If you approve of the anomaly and want to receive similar ones in the future, click the thumbs-up icon in the feedback column. Alternatively, you can right-click the suggestion event/alarm and select *Like*.
+
+   - If you disapprove of the anomaly and do not wish to be notified of similar events in the future, click the thumbs-down icon. Alternatively, you can right-click the suggestion event/alarm and select *Dislike*.
+
+   Once selected, the chosen icon will be pinned in the column as a black icon. If you change your mind, you can click the other option, and only the final feedback will be registered<!--RN 39082-->.
+
+   > [!NOTE]
+   >
+   > - The ![Thumbs-up](~/user-guide/images/Thumbs_Up.png) or ![Thumbs-down](~/user-guide/images/Thumbs_Down.png) icons will only appear for alarms or suggestion events generated by the behavioral anomaly detection feature.
+   > - You can only see the feedback icons you have clicked yourself. Feedback from other users is not visible.
+
+The feedback you provide will be taken into account in the future.
+
+Feedback on active alarms is saved in the user settings and will re-appear after closing and reopening Cube. Feedback on history alarms will disappear after restarting Cube, but it will still be used by SLAnalytics to improve the detection of future anomalies.
```

Comment:
```suggestion
Feedback on active alarms is saved in the user settings and will re-appear after Cube is closed and reopened. Feedback on history alarms will disappear after Cube is restarted, but it will still be used by SLAnalytics to improve the detection of future anomalies.
```